ibis/backends/duckdb/tests/snapshots/test_datatypes/test_cast_to_floating_point_type/float32/out.sql

```diff
@@ -0,0 +1,2 @@
+SELECT
+  CAST('1.0' AS REAL) AS "Cast('1.0', float32)"
```

Comment:
This is produced from `sat.REAL` which compiles to `REAL` and is a valid DuckDB type. Making this return `FLOAT` isn't really worth the additional code since it'll be moot after the port to sqlglot.